GlyssenTests/ProjectTests.cs

```diff
@@ -1,10 +1,205 @@
-﻿using Glyssen.Character;
+﻿using System.IO;
+using System.Linq;
+using System.Threading;
+using Glyssen;
+using Glyssen.Bundle;
+using Glyssen.Character;
+using Glyssen.Quote;
+using GlyssenTests.Bundle;
 using NUnit.Framework;
+using SIL.IO;
+using SIL.ObjectModel;
+using SIL.WritingSystems;
 
 namespace GlyssenTests
 {
-	internal class ProjectTests
+	class ProjectTests
```

Comment:
Running all these tests together as a batch using Resharper in VS causes a series of "The process cannot access the file" messages, and the CreateFromBundle tests hang.